andela/ah-codeofduty
# 161255361 comment on an article

### What does this PR do?
* Implements feature to add comments to an article*
### Description of Task to be completed?
1. Post a comment on an article
2. Get all comments available to a particular article
3. Edit a comment
4. Get a specific comment
5. Delete a comment
### How should this be manually tested?
1. git clone [repo here](https://github.com/andela/ah-codeofduty)
2. create a .env file see .env.example
3. cd ah-codeofduty
4. python manage.py migrate 
5. python manage.py runserver
### Any background context you want to provide?
 - Only authenticated users should be able to comments on articles.
 - Successive comments should be threaded
 - Timestamped comments.
  #### Endpoints
   1. GET api/articles/slug/comment/
   2. POST api/articles/slug/comment/
   3. GET api/articles/slug/comment/article_id/
   4. POST api/articles/slug/comment/article_id/
   5. PUT api/articles/slug/comment/article_id/
   6. DELETE  api/articles/slug/comment/article_id/
### What are the relevant pivotal tracker stories?
 -  [Users should be able to comment on articles](https://www.pivotaltracker.com/story/show/161255361)
### Screenshots (if appropriate)
<img width="777" alt="screen shot 2018-11-08 at 16 09 27" src="https://user-images.githubusercontent.com/17101765/48200636-da1d5d80-e370-11e8-8de7-79e659621b8f.png">


### Questions:
